### PR TITLE
Update to SetupCrossgen 142 - Coherence 25368, add support for timestamped stores for the MSI

### PR DESCRIPTION
**Pull request by mlorbetske**
@mikeharder

### CODE REVIEW

File: build/DependencyVersions.props [hunk 1]
```diff
@@ -24,8 +24,9 @@
 
     <!-- This should either be timestamped or notimestamp as appropriate -->
     <AspNetCoreRuntimePackageFlavor>timestamped</AspNetCoreRuntimePackageFlavor>
-    <AspNetCoreRuntimeVersion>dev-134</AspNetCoreRuntimeVersion>
-    <AspNetCoreCoherenceTimestamp>preview2-25332</AspNetCoreCoherenceTimestamp>
+    <AspNetCoreRuntimeVersion>2.0.0-preview2-139</AspNetCoreRuntimeVersion>
```
Comment (mikeharder) [inline build/DependencyVersions.props]: Is this supposed to be build 142 rather than 139?

**Comment by mlorbetske** [comment]
Looks like requests to MyGet failed for some reason

@dotnet-bot test this please

**Comment by mlorbetske** [comment]
@dotnet-bot test Ubuntu16.04 x64 Debug Build please

**Comment by livarcocc** [comment]
@MattGertz flow of dependencies into the CLI.

**Pull request merged**